tox.ini

```diff
@@ -64,7 +65,8 @@ deps =
 commands_pre =
     oldestdeps: minimum_dependencies romancal --filename requirements-min.txt
     oldestdeps: pip install -r requirements-min.txt
-    devdeps: pip install -r requirements-dev.txt -U --upgrade-strategy eager
+    stdevdeps: pip install -r requirements-dev-st.txt -U --upgrade-strategy eager
+    devdeps: pip install -r requirements-dev-thirdparty.txt -U --upgrade-strategy eager
```

Comment:
This should include the `requirements-dev-st.txt` in addition to the third party ones.